src/backend/lib/queue.js

```diff
@@ -9,6 +9,7 @@ const { logger } = require('../utils/logger');
 
 const client = createRedisClient();
 const subscriber = createRedisClient();
+let redisErrorLogged = false;
```

Comment:
This is named in a very general way, can we do better?  You're keeping track of whether or not we've given a hint about installing Redis if it's not running.  What about `redisNotAvailableLogged`?  Or something that makes sense to you.  I'd also throw a comment above this line explaining its intent.

Reply:
Sure, and done (I settled on `redisConnectionRefusalLogged`): https://github.com/Seneca-CDOT/telescope/pull/553/commits/16b6007a26cab43eccff2a602ce8958282a47213